master/pom.xml

```diff
@@ -75,7 +75,6 @@
   <properties>
     <sakai.version>11-SNAPSHOT</sakai.version>
```

Comment:
Off Topic: Could this just switch to ${project.version} so it just inherits the value from the artifact version?